### ISSUE TITLE
Migrate newQueue.py into SimpleQueue.py using VisualizationApp

### ISSUE THREAD

**Issue by JMCanning78** [JMCanning78/datastructures-visualization#28]
Make a SimpleQueue.py module that subclasses VisualizationApp and
implements the Insert (at rear) and Remove (from front) operations in the
newQueue.py visualization.  We won't be implementing the Deque data structure
operations for this visualization (although we could make a separate Deque
visualization).
Add pointers for the "front" and "rear" of the circular array queue.

(Note: this issue initially was set up to create a module called "Queue.py" but that caused conflicts with the PyCharm debugger.  Now we're calling it "SimpleQueue.py")

**Comment by JMCanning78** [comment]
This issue depends on issue: #16.

**Comment by JMCanning78** [comment]
I will submit a PR shortly that renames newQueue.py into SimpleQueue.py and removes the older queue.py.  After it is merged into `master` please merge `origin/master` into your issue branch (which is named what, by the way?).  I believe that performing the merge will rename the file in your repo.

**Comment by JMCanning78** [comment]
PR #51

**Comment by adinabruce** [comment]
I made a branch for this called Issue28-newqueue-toqueue
